src/main/webapp/script.js

```diff
@@ -12,19 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-google.charts.load('current', {'packages':['corechart']});
-google.charts.setOnLoadCallback(getChart);
-
 const CHART_WIDTH = 800;
 const CHART_HEIGHT = 400;
+const HIGHEST_SCORE = 1.0;
+const LOWEST_SCORE = -1.0;
+const INTERVAL = 0.2;
```

Comment:
Just a thought: maybe we can pass this INTERVAL variable later into the YoutubeServlet.java; we can make this variable

Reply:
Spoke offline as group & opened up new issue to deal with variables across multiple files